luda-editor/luda-editor-client/src/app/sequence_list/mod.rs

```diff
@@ -108,31 +121,103 @@ impl Entity for SequenceList {
                         }
                     });
                 }
+                SequenceListEvent::SequenceTitlesLoadStateUpdateEvent { state } => {
+                    if let Some(old_state) = &self.sequence_titles_load_state {
+                        if old_state.started_at > state.started_at {
+                            return;
+                        }
+                    }
+
+                    self.sequence_titles_load_state = Some(state.clone());
+                }
+                SequenceListEvent::SequenceTitlesLoadEvent => {
```

Comment:
I think it's too big code to put in `update`. Would you like to move this logic into other fn with nice name and just call it?

Reply:
How about `fn handle_*_event()`?
Does it sound like a bad name?

Reply:
`fn handle_*_event()` was bad name I think.
Changed to name that represents what a function does with 4653fbf